README.md

```diff
@@ -35,8 +35,8 @@ jobs:
     runs-on: 'ubuntu-20.04'
     # Declare outputs for next jobs
     outputs:
-      apifiles_changed: ${{ steps.check_files_changed.outputs.apifile_updates }}
-      action_changed: ${{ steps.check_files_changed.outputs.action_updates }}
+      apifiles_changed: ${{ steps.check_apifile_changed.outputs.apifile_updates }}
+      action_changed: ${{ steps.check_apifile_changed.outputs.action_updates }}
```

Comment:
Can we update the id to be `check_files_changed` as that was my original intention :-)